### ISSUE TITLE
Media search engine optimization

### ISSUE THREAD

**Issue by ebraker** [ArctosDB/arctos#4895]
@dustymc Do you have any insights as to why media images don't show up in a Google Search?

For instance, searching Google for "Xenosaurus rackhami" "Xenosaurus rackhami rackhami" or the exact text of the media_description
"UCM Herp 19028 Xenosaurus rackhami dorsal" (media object 10680067) turns up nothing for this specimen:

https://web.corral.tacc.utexas.edu/arctos/ucm/oMeso_Herps/2021-11-09/UCM_Herp_19028_Xenosaurus_rackhami_dorsal.jpg
https://arctos.database.museum/guid/UCM:Herp:19028

Some of our project images are literally the only images available on the web for certain rare taxa. But they are not discoverable to public audiences that don't know about GBIF...

I'm not sure how we could optimize search results without paying Google but I'm curious why some Arctos media don't float to the top of the algorithm if they are in fact novel and exact string matches?

**Comment by ebraker** [comment]
I also just queried "Xenosaurus rackhami" on a basic Google search, which only has 996 results. None of these involve Arctos (there is one GBIF result) - neither the two catalog record pages or the taxonomy page.

**Comment by dustymc** [comment]
No, I don't really understand how Google works or why/how they do what they do. I've got their crawler ips safelisted so I'm relatively sure nothing's missing because I'm blocking it, beyond that it looks like magic from here.

There's some chance the exclusion is partially related to https://github.com/ArctosDB/arctos/issues/2893 - they seem to use mostly mobile bots, Arctos doesn't deal with that very well.

https://github.com/ArctosDB/arctos/discussions/4343 might be related - Arctos media is a wild mix of formats and EXIF and etc., maybe Google would do better if it was more standardized.
